.github/workflows/vulnerability-scanning-on-repo.yml

```diff
@@ -65,15 +65,37 @@ jobs:
         uses: actions/download-artifact@v4
         with:
           name: ${{ github.run_number }}-sbom.json
-      - name: Scan SBOM
+      - name: Scan SBOM, report as table file
         uses: aquasecurity/trivy-action@master
         with:
-          exit-code: '0'
-          ignore-unfixed: true
-          # https://github.com/aquasecurity/trivy-action/pull/314
-          scan-ref: ${{ env.BOM_FILE_PATH }}
+          exit-code: '0' # always pass
+          scan-type: 'sbom'
+          scan-ref: ${{ env.BOM_FILE_PATH }} # Need; See /aquasecurity/trivy-action/pull/314
+          scanners: 'vuln'
+          format: 'table'
+          output: trivy-sbom-vuln-report.txt
```

Comment:
If you set both `format` and `output` then the output goes to the designated output file.  If you wanted to see this on stdout then you'd have to run the action separately with only `format` set

Reply:
Hm. So that is what I expected. Perhaps my testing method was wrong (HIGH,CRITICAL doesn't output anything )


Hm, side note - maybe HIGH,CRITICAL is actually broken. I'll check both.